src/ucs/config/parser.c

```diff
@@ -1387,6 +1387,33 @@ ucs_config_parser_set_default_values(void *opts, ucs_config_field_t *fields)
     return UCS_OK;
 }
 
+static int
+ucs_config_name_match_no_check(const char *name, const char *pattern)
+{
+    return !fnmatch(pattern, name, 0);
+}
```

Comment:
IMO this wrapper is redundant

Reply:
this wrapper was added:
1) to improve readability of `ucs_config_prefix_name_match`
2) to have external `fnmatch` call and matching ligic in single place since `fnmatch` has a perameter which changes its behaviour (can be changed in one place and missed in other one)

basically, all above is minor polishing, so the wrapper can be removed if you are disagree